### PR TITLE
code cleanups

### PR DESCRIPTION
**Pull request by xhulianoThe1**
## Proposed changes
Small code cleanups. 

## Types of changes

What types of changes does your code introduce to **CausalML**?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/uber/causalml/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/uber/causalml)
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc. This PR template is adopted from [appium](https://github.com/appium/appium).

### CODE REVIEW

**Comment by xhulianoThe1** [comment]
@jeongyoonlee updated!

File: causalml/inference/meta/tmle.py [hunk 1]
```diff
@@ -126,7 +126,7 @@ def __repr__(self):
             self.__class__.__name__, self.model_tau.__repr__(), self.cv
         )
 
-    def estimate_ate(self, X, treatment, y, p, segment=None, return_ci=False):
```
Comment (jeongyoonlee) [inline causalml/inference/meta/tmle.py:129]: Ideally, it should follow the signature from `causalml.inference.meta.base.BaseLearner.estimate_ate()`. Can you update `TMLELearner` to inherit from `BaseLearner` and keep the function signatures consistent with other meta learners?

**Review by jeongyoonlee — COMMENTED** [review]
Thanks for the contribution! LGTM except one for `tmle.py`. If you'd like to keep this PR simple without touching `tmle.py`, that's fine too.

File: causalml/inference/meta/tmle.py (hunk 1 above)
Comment (xhulianoThe1) [reply causalml/inference/meta/tmle.py]: Will keep this pull request simple. 
Will open a new one for a deeper dive into `tmle.py`

**Review by jeongyoonlee — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**